README.md

```diff
@@ -44,6 +45,13 @@ There are existing proposals to allow different domains to declare an affiliatio
 
 The file [`quirks/change-password-URLs.json`](quirks/change-password-URLs.json) contains a JSON object mapping domains to URLs where users can change their password. This is the quirks version of the [Well Known URL for Changing Passwords](https://wicg.github.io/change-password-url/index.html). If a website adopts the Change Password URL, it should be removed from this list.
 
+### Websites that Append 2nd Factor to Password
+
+The file [`quirks/websites-that-append-2fa-to-password.json`](quirks/websites-that-append-2fa-to-password.json) contains a JSON array of domains which use a two-factor authentication scheme where the user must append a generated code to their password when signing in. Should be used to:
+
+1. Prevent auto-submission of login forms to allow the 2FA token to be added by the user.
+1. Suppress "remember new password" prompts when the submitted password starts with the already stored password.
```

Comment:
I suggest, “Suppress prompting to update a saved password when the submitted password has a prefix of the already-stored password."

Reply:
And, looking at this now, I bet these two points could be made as a single sentence or two sentences, without the load of a list.

My attempt:

```
The file [`quirks/websites-that-append-2fa-to-password.json`](quirks/websites-that-append-2fa-to-password.json) contains a JSON array of domains which use a two-factor authentication scheme where the user must append a generated code to their password when signing in. This list of websites could be used to prevent auto-submission of login forms, allowing the user to append the 2FA code without frustration. It can also be used to suppress prompting to update a saved password when the submitted password is prefixed by the already-stored password.
```